### DISCUSSION TITLE
How to patch scaled objects?

### DISCUSSION THREAD

**Discussion by wzhiana** [kedacore/keda#6151] [Q&A / Need Help] [answered]
I'm trying to patch scaled object like:
`kubectl patch scaledobject -n namespace scaled_project -p '{"spec":{"triggers":[{"metadata":{"value":80},"metricType":"Utilization","type":"cpu"},{"metadata":{"value":80},"metricType":"Utilization","type":"memory"}]}}'`
and it fails:
`Error from server (UnsupportedMediaType): the body of the request was in an unknown format - accepted media types include: application/json-patch+json, application/merge-patch+json, application/apply-patch+yaml`

why? how to workaround that?

**Comment by rickbrouwer** [comment, marked as answer]
You need to specify the patch type (`--type`).
```
kubectl patch scaledobject -n namespace scaled_project --type json -p '[{"op":"replace","path":"/spec/triggers","value":[{"metadata":{"value":"80"},"metricType":"Utilization","type":"cpu"},{"metadata":{"value":"80"},"metricType":"Utilization","type":"memory"}]}]'
```
